ios/Podfile

```diff
@@ -26,20 +24,8 @@ target 'ledgerlivemobile' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
+  use_flipper!({ 'Flipper-Folly' => '2.3.0' })
```

Comment:
workaround for ['event2/event-config.h file' not found error](https://stackoverflow.com/a/66055723)